toolchain-fitnesse-plugin-test/pom.xml

```diff
@@ -0,0 +1,255 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>toolchain-fitnesse-plugin-package</artifactId>
+
+        <groupId>nl.praegus</groupId>
+        <version>2.0.3-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>Toolchain-Fitnesse-Plugin-Test</artifactId>
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+        <fitnesse.version>20200404</fitnesse.version>
+        <hsac.fixtures.version>4.14.0</hsac.fixtures.version>
+        <toolchain.fixtures.version>1.21</toolchain.fixtures.version>
+        <toolchain-plugin.version>2.0.3</toolchain-plugin.version>
```

Comment:
Hier kun je ${project.version} in zetten, dan pakt ie de snapshot versie op die je wil testen.